assets/common/src/deploy.py

```diff
@@ -260,12 +261,17 @@ def main():
     args = parse_args()
     ml_client = get_mlclient()
     # get registered model id
-    if args.registration_details:
+    if args.registration_details_folder and \
+            os.path.exists(args.registration_details_folder/ComponentVariables.REGISTRATION_DETAILS_JSON_FILE):
```

Comment:
Please use something like below for readibility: 

```
registration_details_file = args.registration_details_folder / ComponentVariables.REGISTRATION_DETAILS_JSON_FILE
if registration_details_file.exists():
   ....

Reply:
Done